rita/src/rita_common/dashboard/mod.rs

```diff
@@ -42,16 +48,22 @@ impl Handler<GetOwnInfo> for Dashboard {
     type Result = ResponseFuture<OwnInfo, Error>;
 
     fn handle(&mut self, _msg: GetOwnInfo, _ctx: &mut Self::Context) -> Self::Result {
-        Box::new(
-            PaymentController::from_registry()
-                .send(GetOwnBalance {})
-                .from_err()
-                .and_then(|res| {
-                    Ok(OwnInfo {
-                        balance: res?,
-                        version: env!("CARGO_PKG_VERSION").to_string(),
-                    })
-                }),
-        )
+        #[cfg(feature = "guac")]
```

Comment:
this is cleaner than that it replaces, we should try and use this pattern more often.

Reply:
BTW for blocks of feature dependent code wouldn't it be more readable to do `if cfg!(...)` instead?